numpy/core/fromnumeric.py

```diff
@@ -1895,6 +1905,22 @@ def sum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
                          out=out, **kwargs)
 
 
+def product(a, axis=None, dtype=None, out=None, keepdims=np._NoValue, initializer=np._NoValue):
```

Comment:
This function should be removed - it's now defined with `*args, **kwargs` at the bottom of the file.